sdcm/cluster.py

```diff
@@ -585,8 +585,8 @@ def install_scylla_bench(self):
             sb_version = f'heads/{sb_version}'
         self.remoter.sudo(shell_script_cmd(f"""\
             rm -rf /usr/local/go
-            rm -f /tmp/sb_install || true
-            mkdir /tmp/sb_install
+            rm -rf /tmp/sb_install || true
+            mkdir -p /tmp/sb_install
```

Comment:
It's redundant to use `-p` switch because the directory should be deleted at this point. But it doesn't matter, actually.

Reply:
yes, but what failed the test was the `mkdir`, so i prefer to "ignore" its errors, although now the `rm` will always succeed